_includes/header.html

```diff
@@ -37,6 +35,16 @@
         </li>
         <li><a href="{{ site.baseurl }}/blog">Blog</a></li>
       </ul>
+      <ul class="nav navbar-nav navbar-right">
+      <li class="dropdown">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project <span class="caret"></span></a>
```

Comment:
Orthogonal: not sure that the current menu organization makes a lot of sense. But, let's not block this PR on that.